db/migrate/20190126141332_create_core_emails.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCoreEmails < ActiveRecord::Migration[5.2]
+  def change
+    create_table :core_emails do |t|
```

Comment:
@RichardLaskowski I missed this until after I merged it in. I'll do a hotfix for it, but don't forget we need to have the code here that uses UUIDs.

While I am modifying the existing migration keep in mind we won't be doing this long. However since we are doing this we will have to drop, create, and migrate each time we pull down the project/core branch. However once this core is done we will modify existing tables through a new migration and no longer edit previous migrations